Sources/Tests/FSharpWrapperUnitTests/MatrixOperatorsTest.fs

```diff
@@ -0,0 +1,63 @@
+﻿module MatrixOperatorsTest
+
+open AngouriMath.FSharp.Core
+open AngouriMath.FSharp.Functions
+open Xunit
+open AngouriMath
+open AngouriMath.FSharp.MatrixOperators
+(*
+
+ Matrix and matrix operators
+
+*)
+
+[<Fact>]
+let ``M-M +`` () =
```

Comment:
The hyphen can be easily misidentified as minus. Use e.g. `M+M` instead. Ditto for each "parentheses not needed".